src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs

```diff
@@ -347,53 +348,86 @@ public void Dispose()
                 Assert.Equal(new byte[bufferSize], buffer);
 
                 context.Response.Close();
+                await clientTask;
             }
         }
 
         [Fact]
         public async Task CanSeek_Get_ReturnsFalse()
         {
-            HttpListenerRequest response = await _helper.GetRequest(chunked: true);
```

Comment:
You are right, the ContextHelper is no longer needed for this test class. I will fix this in my new PR.